### PR TITLE
builder, gh actions: add centos-8-stream build

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: builder-support/dockerfiles/Dockerfile.target.centos-8 [hunk 1]
```diff
@@ -12,7 +12,13 @@ FROM amd64/centos:8 as dist-base
 @IF [ ${BUILDER_TARGET} = centos-8-arm64 ]
 FROM arm64v8/centos:8 as dist-base
 @ENDIF
+
 ARG BUILDER_CACHE_BUSTER=
+
+@IF [[ $BUILDER_TARGET = *stream* ]]
```
Comment (omoerbeek) [inline builder-support/dockerfiles/Dockerfile.target.centos-8:18]: Above you use "centos-8-stream"

Comment (omoerbeek) [reply builder-support/dockerfiles/Dockerfile.target.centos-8:18]: err, missed the asterisks... but I do think `centos-*-stream` is better

Comment (Habbie) [reply builder-support/dockerfiles/Dockerfile.target.centos-8]: done!

**Review by omoerbeek — COMMENTED** [review]
Looks good to me as far as  I can judge (which isn't very far ;-)

**Review by pieterlexis — APPROVED** [review]

**Comment by Habbie** [comment]
Rebased.

**Pull request merged**